packages/office-addin-debugging/src/start.ts

```diff
@@ -203,11 +270,13 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
         // live reload can only be enabled for the desktop app type
         // when using proxy debugging and the packager
         const canEnableLiveReload: boolean = isDesktopAppType && isProxyDebuggingMethod && !!packagerCommandLine;
+        // only enable live reload if it can be enabled
+        const liveReload = enableLiveReload && canEnableLiveReload;
+
         let packagerPromise: Promise<void> | undefined;
         let devServerPromise: Promise<void> | undefined;
 
-        // only enable live reload if it can be enabled
-        enableLiveReload = enableLiveReload && canEnableLiveReload;
+
```

Comment:
I think there is an extra blank line or two here that you can remove. You should just have one blank line between variable declarations and the code after.